packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -236,6 +236,12 @@ typedef TransformCallback = Matrix4 Function(double animationValue);
 /// The [onTransform] callback computes a [Matrix4] from the animated value, it
 /// is called every time the [animation] changes its value.
 ///
+/// {@tool dartpad}
+/// The following code implements the [MatrixTransition]:
```

Comment:
Can you please describe what the example does? 

Take a look at other example descriptions for ideas.

Reply:
I added some more detail. PTAL.